README.md

```diff
@@ -115,4 +115,13 @@ Please create a GitHub issue to report an issue or ask questions.
 ## Contributing
 
 Contributions to the OpenCL ICD Loader are welcomed and encouraged.
-You will be prompted with a one-time "click-through" CLA dialog as part of submitting your pull request or other contribution to GitHub.
\ No newline at end of file
+You will be prompted with a one-time "click-through" CLA dialog as part of submitting your pull request or other contribution to GitHub.
+
+## Table of Debug Environment Variables
+
+The following debug environment variables are available for use with the OpenCL ICD loader:
+
+| Environment Variable              | Behavior            |  Example Format      |
+|:---------------------------------:|---------------------|----------------------|
+| OCL_ICD_FILENAMES                 | Specifies a list of additional ICDs to load.  The ICDs will be enumerated first, before any ICDs discovered via default mechanisms. | `export OCL_ICD_FILENAMES=libVendorA.so:libVendorB.so`<br/><br/>`set OCL_ICD_FILENAMES=vendor_a.dll;vendor_b.dll` |
+| OCL_ICD_VENDORS                   | On Linux and Android, specifies a directory to scan for ICDs to enumerate in place of the default `/etc/OpenCL/vendors'. |  `export OCL_ICD_VENDORS=/my/local/icd/search/path` |
```

Comment:
If we allow paths in OCL_ICD_ENTRIES as suggested above, I think this is not needed anymore?

Reply:
No, I think we should support both options as they have different uses:

`OCL_ICD_VENDORS` still uses the .icd files, but it reads them from a different directory than `/etc/OpenCL/vendors`.  It is only valid on Linux and Android (and probably OSX as well, when using the ICD loader).

`OCL_ICD_FILENAMEs` is used to explicitly load the specified ICDs, without requiring any .icd files.  It works on all OSes.